### DISCUSSION TITLE
Google HTML/CSS Style Guide

### DISCUSSION THREAD

**Discussion by brunodevel** [frontendbr/forum#333] [Discussão]
Galera, nesse link tem um style guide bem bacana para HTML e CSS:
https://google.github.io/styleguide/htmlcssguide.xml

Alguns pontos eu não conhecia, mas tem um específico que me deixou surpreso e eu não tinha nem ideia de que era válido: _HTML Style Rules > Optional Tags_.

Já escreveram algo omitindo as tags dessa forma?

**Comment by marcoaure** [comment]
muito bom o link, obrigado pelo share

Atenciosamente, _Marco Aurélio_.  _Perfil http://marcoaure.net *
Analista-Desenvolvedor de sistemas e especialista: *Php__, *_Mysql**, *
*Html5**, **CSS3**, **JQuery**, **Front *e *Back-End**.*

Em 19 de outubro de 2016 23:26, Bruno Silva notifications@github.com
escreveu:

**Comment by akfzambrana** [comment]
Eu nunca fiz isso de omitir as tags. Honestamente não curti kkkk

Eu utilizo este styleguide pra trabalhar http://codeguide.co/ mas tb curto este aqui http://workmanship.io/ que é praticamente igual, so tem algumas "perfumarias" diferentes

**Comment by fititnt** [comment]
O "Optional Tags" é algo que cheguei a escrever um rascunho no medium.com criticando isso mas não publiquei. Estou na correria aqui, mas outra hora comento.

E eu não recomendo isso.

**Comment by brunodevel** [comment]
Boa, @fititnt 
Quando postar compartilha com a gente.

Também não curti a remoção dessas tags, talvez por estar muito acostumado com o padrão.

Tem até uma observação no guide sobre isso:

> This approach may require a grace period to be established as a wider guideline as it’s significantly different from what web developers are typically taught.

**Comment by fititnt** [comment]
**TL;DR: não recomendo. Sites internacionais, não podem omitir no mínimo algumas tags, como a `<html lang="pt">`**

---

Pessoal, aqui tem o rascunho em inglês https://medium.com/@fititnt/dear-google-stop-suggestion-omiting-all-optiomal-tags-on-your-html-styleguide-for-a-better-a11y-1189d8efe79f

Se alguém tiver conta lá, pode fazer comentários, senão, pode comentar por aqui mesmo. 

Eu quero ver se faço aconselhamento pra alguem fazer um Pull Request no repositório do Google e ser aceito de primeira, mas surgiu outras coisas na @WebIWG e acabei deixando rascunho parado.
